CHANGELOG.md

```diff
@@ -21,6 +21,7 @@
   specified instance, running within a cgroup.
 - Add `--sparse` flag to `overlay create` command to allow generation of a
   sparse ext3 overlay image.
+- Support for `DOCKER_HOST` parsing when using `docker-daemon://`
```

Comment:
Also mention "DOCKER_USERNAME` and `DOCKER_PASSWORD` now supported without the `SINGULARITY_` prefix.